code/datums/mind.dm

```diff
@@ -177,6 +175,22 @@
 			. += "</a>"
 		. += "<br>"
 
+/**
+ * Gets every objective this mind owns, including all of those from any antag datums they have, and returns them as a list.
+ */
+/datum/mind/proc/get_all_objectives()
+	var/list/all_objectives = list()
+
+	for(var/antag in antag_datums)
+		var/datum/antagonist/A = antag
+		all_objectives |= A.objectives // Add all antag datum objectives.
```

Comment:
Are objectives ever shared between antag datums or in the mind? If not then the `|=` won't be needed here. Though that needs to be double checked